src/java/com/unifina/feed/AbstractFeedProxy.java

```diff
@@ -75,53 +67,35 @@ public void receive(Message parsedMsg) {
 			throw new RuntimeException(e);
 		}
 	}
-	
-	private void processAndQueue(long counter, MessageClass msg, boolean checkAge) {
-		if (counter != expected) {
-			throw new IllegalArgumentException("Tried to process messages in invalid order! Counter: " + counter + ", expected: " + expected);
-		} else {
-			expected++;
-			FeedEvent<MessageClass, EventRecipientClass>[] events = process(msg);
-
-			if (firstRealQueue == null && checkAge) {
-				firstRealQueue = counter;
-			}
 
-			if (events != null) {
-				// Warn about old events
-				if (checkAge && events.length > 0) {
```

Comment:
The `checkAge` logic and logged warnings are important and shouldn't be removed! It's currently the only way to see from the logs if there's delay/backpressure in the message queue.

Sure it could be more sophisticated, but...